Assets/Scripts/Buddy/BuddyStats.cs

```diff
@@ -66,6 +66,18 @@ enum HappinessState
 	[SerializeField] float _emoteRoutineWait = 0f;
 	Coroutine _currentEmoteRoutine = null;
 
+	[Header( "Debug Settings" )]
+	[SerializeField] bool _disableStatDecrease = false;
+
+	[SerializeField] ParticleSystem _particles;
+	Renderer _particlesRenderer;
+	[SerializeField] ParticleSystem _deadParticleSystem;
```

Comment:
I'm not sure if the particles would properly outlive the particle system, but I do know for certain that you can't change the color over time, which we would need to do. The emotes fade in and out but if the death did that it would look bad. So I don't think we can do this with one particle system.